WordPress/Classes/ViewRelated/Stats/Insights/SiteStatsInsightsViewModel.swift

```diff
@@ -75,6 +75,8 @@ class SiteStatsInsightsViewModel: Observable {
     }
 
     func startFetchingPeriodOverview() {
+        self.lastRequestedPeriod = StatsPeriodUnit.day
```

Comment:
This change may not be needed.

`self.lastRequestedPeriod = StatsPeriodUnit.day` is set in `init()` and never changed. I tested with and without this change and couldn't find it affecting the outcome.

Reply:
Thanks for looking at this with a fresh set of eyes! I agree as well that I don't think this is needed.  I've removed